1.32.0 release notes (#4386)
kiegroup/kie-docs: doc-content/kogito-docs/src/main/asciidoc/configuration/chap-kogito-configuring.adoc

NOTE: Data Index Service with PostgreSQL does not support indexing domain-specific data. The GraphQL schema only exposes queries for `ProcessInstances`, `UserTaskInstances`, and `Jobs`.

> ==== PostgreSQL database setup for the {PRODUCT} Data Index Service

Reviewer: ```suggestion
==== PostgreSQL database setup for the {PRODUCT} data index service
```

Author: @kaldesai I believe we reference the service name in upper case, like https://docs.kogito.kie.org/latest/html_single/#con-data-index-service_kogito-configuring ?

> {PRODUCT} provides two ways to manage the database schema, either via manually applying the provided DDL https://repository.jboss.org/org/kie/kogito/kogito-ddl/{COMMUNITY_VERSION_FINAL}/[kogito-ddl-{COMMUNITY_VERSION_FINAL}-db-scripts.zip], or via https://flywaydb.org/[Flyway] integration. This integration is done via Quarkus Flyway and the following https://quarkus.io/guides/flyway[guide] can give extra details about how to use it. In a nutshell, to create the required tables for Data Index on an empty database, the following property should be added `quarkus.flyway.migrate-at-start=true`.

Reviewer: ```suggestion
{PRODUCT} provides two ways to manage the database schema, either via manually applying the provided DDL https://repository.jboss.org/org/kie/kogito/kogito-ddl/{COMMUNITY_VERSION_FINAL}/[kogito-ddl-{COMMUNITY_VERSION_FINAL}-db-scripts.zip], or via https://flywaydb.org/[Flyway] integration. This integration is done via Quarkus Flyway and the following https://quarkus.io/guides/flyway[guide] can give extra details about how to use it. In a nutshell, to create the required tables for data index on an empty database, you must add the `quarkus.flyway.migrate-at-start=true` property.
```
